testing/dart/channel_buffers_test.dart

```diff
@@ -160,7 +160,7 @@ void main() {
       didCallCallback = true;
     }
     void twoCallback(ByteData responseData) {
-      throw TestFailure('wrong callback called');
+      throw Exception('wrong callback called');
```

Comment:
nit: it's probably trivial to define `TestException` in `package:litetest` for additional compatibility, and will give us clearer signal on failures that it was a test failure and not some random exception.

Reply:
Changed to `fail()`.